staging/src/k8s.io/dynamic-resource-allocation/resourceslice/resourceslicecontroller.go

```diff
@@ -155,7 +225,8 @@ func (c *Controller) Stop() {
 // Update sets the new desired state of the resource information.
 //
 // The controller takes over ownership, so these resources must
-// not get modified after this method returns.
+// not get modified after this method returns. [DriverResources.DeepCopy]
+// can be used by the caller to clone some existing instance.
```

Comment:
Likewise here -- why can't we / don't want to just do the `DeepyCopy()` internally?

Reply:
https://github.com/kubernetes/kubernetes/pull/128275#discussion_r1822302950 - will add it.